lib/chef/knife/kvm_vm_create.rb

```diff
@@ -140,6 +144,15 @@ def read_and_validate_params
           exit 1
         end
 
+        if (config[:location].nil? &&
```

Comment:
Yeah I'm on the fence. I don't really like either option for readability... I don't feel super strongly about it though. I'll probably leave it unless you really object @patrick-wright